pkgs/applications/misc/jrnl/default.nix

```diff
@@ -1,30 +1,44 @@
 { lib
+, fetchFromGitHub
 , python3
 }:
 
 with python3.pkgs;
```

Comment:
Maybe we can move this to the place it is needed.

Reply:
Changed. Thanks for the feedback.